iroha/src/lib.rs

```diff
@@ -83,9 +83,36 @@ impl Iroha {
         let (wsv_blocks_sender, wsv_blocks_receiver) = sync::channel(100);
         let (kura_blocks_sender, kura_blocks_receiver) = sync::channel(100);
         let (message_sender, message_receiver) = sync::channel(100);
-        let world_state_view = Arc::new(RwLock::new(WorldStateView::new(Peer::new(
+        let domain_name = "global".to_string();
+        let mut asset_definitions = HashMap::new();
+        let asset_definition_id = permission::permission_asset_definition_id();
+        asset_definitions.insert(
+            asset_definition_id.clone(),
+            AssetDefinition::new(asset_definition_id.clone()),
+        );
+        let account_id = AccountId::new("root", &domain_name);
+        let asset_id = AssetId {
+            definition_id: asset_definition_id,
+            account_id: account_id.clone(),
+        };
+        let asset =
+            Asset::with_permission(asset_id.clone(), ("anything".to_string(), "".to_string()));
+        let mut account =
+            Account::new(&account_id.name, &account_id.domain_name, config.public_key);
+        account.assets.insert(asset_id, asset);
+        let mut accounts = HashMap::new();
+        accounts.insert(account_id, account);
+        let domain = Domain {
+            name: domain_name.clone(),
+            accounts,
+            asset_definitions,
+        };
+        let mut domains = HashMap::new();
+        domains.insert(domain_name, domain);
+        let world_state_view = Arc::new(RwLock::new(WorldStateView::new(Peer::with_domains(
```

Comment:
I think this should be in genesis block. Right now it seems to be not in blockchain at all.

If not in genesis block, then maybe at least take these values from configuration?

Reply:
@eadventurous totally agree with you, let's put in on an additional issue for a genesis block functionality (maybe some things we will move to another config, but we definitely need genesis)
![image](https://user-images.githubusercontent.com/5635032/83855196-c8589380-a728-11ea-913b-4cda402501f6.png)
Jira is not working right now, I will hold PR till issue creation

Reply:
https://jira.hyperledger.org/secure/RapidBoard.jspa?rapidView=264&view=detail&selectedIssue=IR-756